examples/Azure_IoT_Hub_ESP32/Azure_IoT_Hub_ESP32.ino

```diff
@@ -296,15 +297,10 @@ static void establishConnection()
 
 static void getTelemetryPayload(az_span payload, az_span* out_payload)
 {
-  az_span original_payload = payload;
+  // You can generate the json using any lib you want. Here it's hardcoded for simplicity.
```

Comment:
Actually, this is not hardcoded. You're still taking the message count to publish, but I get your point. I'd say something like:
"// You can generate the JSON using any lib you want. Here we're showing how to do it manually, for simplicity"

Reply:
Sure. I can update that comment.